src/pages/Groups.tsx

```
@@ -85,6 +85,34 @@
+    // Sort by recent activity (most active communities first)
+    const getDTag = (community: any) => community.tags?.find((tag: any) => tag[0] === "d");
+    const getCommunityId = (community: any) => `34550:${community.pubkey}:${getDTag(community) ? getDTag(community)![1] : ""}`;
+    
+    const aId = getCommunityId(a);
+    const bId = getCommunityId(b);
+    
+    const aStats = communityStats ? communityStats[aId] : undefined;
+    const bStats = communityStats ? communityStats[bId] : undefined;
+    
+    // Calculate activity score based on posts and participants
+    const getActivityScore = (stats: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)